lib/Widget/dashboard/dashboard_Table.dart

```diff
@@ -193,6 +195,25 @@ class _DashboardTableState extends State<DashboardTable> {
       "WtHt": 56,
     },
   ];
+  splitdiastolic(String bloodpressure) {
```

Comment:
Consider using camel-case

Reply:
@PormpimonPang

Reply:
Can't use camel-case because BloodPressure ("120/85") is not alphabet
but I fixed it more for reused the code for split the bloodpressure.

Reply:
Resolve in 143d0ec